src/modules/function.js

```diff
@@ -0,0 +1,7 @@
+export default class myObject {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
```

Comment:
Since you only made use of the myObject class once in the addtodolist function you should create it in the index.js file to follow the yagni principle